chart/kafka-connector/templates/rbac.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
```

Comment:
Can this be installed conditionally?

Reply:
Yes, it can. I is only required when async invocations are enabled and the `async.maxInflight` > 0

Reply:
This is now conditional.